src/wallet/rpcwallet.cpp

```
@@ -3332,7 +3332,7 @@ UniValue rescanblockchain(const JSONRPCRequest& request)
         }
     }
 
-    CBlockIndex *stopBlock = pwallet->ScanForWalletTransactions(pindexStart, pindexStop, reserver, true);
+    const CBlockIndex* stopBlock = pwallet->ScanForWalletTransactions(*pindexStart, pindexStop, reserver, true);
```

`Genesis` can return `nullptr` - maybe throw on that above rather than possibly get a null dereference error?

---

Good point. Fixed